src/egon/data/datasets/gas_neighbours.py

```diff
@@ -0,0 +1,1520 @@
+        .drop(columns=["Parameter", "Year", "Case"])
+    )
+    df_conv_2040_average = (
+        df[
+            (df["Parameter"] == "Conventional")
+            & (df["Year"] == 2040)
+            & (df["Case"] == "Average")
+        ]
+        .rename(columns={"Value": "Value_conv_2040_average"})
+        .drop(columns=["Parameter", "Year", "Case"])
+    )
+    df_bioch4_2040 = (
+        df[
+            (df["Parameter"] == "Biomethane")
+            & (df["Year"] == 2040)
+            & (
+                df["Case"] == "Peak"
+            )  # Peak and Average have the same valus for biogas production in 2030 and 2040
+        ]
+        .rename(columns={"Value": "Value_bio_2040"})
+        .drop(columns=["Parameter", "Year", "Case"])
+    )
+    df_2040 = pd.concat(
+        [df_conv_2040_peak, df_conv_2040_average, df_bioch4_2040, lng], axis=1
+    ).fillna(0)
+    df_2040 = df_2040[
+        ~(
+            (df_2040["Value_conv_2040_peak"] == 0)
+            & (df_2040["Value_bio_2040"] == 0)
+            & (df_2040["LNG max_cap_store2pipe_M_m3_per_d (in GWh/d)"] == 0)
+        )
+    ]
+    df_2040["Value_conv_2040"] = (
+        df_2040["Value_conv_2040_peak"]
+        + df_2040["LNG max_cap_store2pipe_M_m3_per_d (in GWh/d)"]
+    )
+    df_2040["CH4_2040"] = (
+        df_2040["Value_conv_2040"] + df_2040["Value_bio_2040"]
+    )
+    df_2040["ratioConv_2040"] = (
+        df_2040["Value_conv_2040_peak"] / df_2040["CH4_2040"]
+    )
+    df_2040["e_nom_max_2040"] = (
+        df_2040["Value_conv_2040_average"] + df_2040["Value_bio_2040"]
+    )
+    df_2040 = df_2040.drop(
+        columns=[
+            "LNG max_cap_store2pipe_M_m3_per_d (in GWh/d)",
+            "Value_conv_2040_average",
+            "Value_conv_2040_peak",
+        ]
+    )
+
+    # Conversion GWh/d to MWh/h
+    conversion_factor = 1000 / 24
+
+    df_2035 = pd.concat([df_2040, df_2030], axis=1)
+    df_2035 = df_2035.drop(
+        columns=[
+            "Value_conv_2040",
+            "Value_conv_2030",
+            "Value_bio_2040",
+            "Value_bio_2030",
+        ]
+    )
+    df_2035["cap_2035"] = (df_2035["CH4_2030"] + df_2035["CH4_2040"]) / 2
+    df_2035["e_nom_max"] = (
+        ((df_2035["e_nom_max_2030"] + df_2035["e_nom_max_2040"]) / 2)
+        * conversion_factor
+        * 8760
+    )
+    df_2035["ratioConv_2035"] = (
+        df_2035["ratioConv_2030"] + df_2035["ratioConv_2040"]
+    ) / 2
+    grouped_capacities = df_2035.drop(
+        columns=[
+            "ratioConv_2030",
+            "ratioConv_2040",
+            "CH4_2040",
+            "CH4_2030",
+            "e_nom_max_2030",
+            "e_nom_max_2040",
+        ]
+    ).reset_index()
+
+    grouped_capacities["cap_2035"] = (
+        grouped_capacities["cap_2035"] * conversion_factor
+    )
+
+    # Add generator in Russia
+    grouped_capacities = grouped_capacities.append(
+        {
+            "cap_2035": 100000000000,

```

Comment:
Where do these numbers come from? Would be helpful to mention the source here.

Reply:
We consider Russia as a infinite production source, so this value is an arbitrary high value. Should I comment it to explain it?

Reply:
Could you then replace the arbitrary high values by inf and leave a short comment in the code?

Reply:
I let a float value for the p_nom parameter of the russian generator because we noticed that:

- Infinite is not correctly written in the database. When trying (with all the imaginable and even the wildest ways) to write inf in p_nom, the default value (0) is passed to the parameter.
- PyPSA doesn't deal with and infinite value in p_nom for a generator.